src/theme.rs

```diff
@@ -49,7 +52,7 @@ impl Theme {
             real
         } else {
             match config_file::Config::config_file_path() {
-                Some(p) => p.join("themes").join(real),
```

Comment:
We can probably support both `~/.config/lsd/themes/<name>.yaml` and `~/.config/lsd/colors.yaml` initially for a while. Dropping it without any warning might not be the best idea. Also, it will be `~/.config/lsd/colors.yaml` and not `~/.config/lsd/<name>.yaml` essentially making `config.color.theme`  unnecessary.

Reply:
Oops, sorry for mistakenly changing the location, I have changed it back to the origin one and added a TODO sign, and a new issue https://github.com/Peltoche/lsd/issues/749 to track.